XenAdmin/SettingsPanels/NRPEEditPage.cs

```diff
@@ -75,13 +77,12 @@ public string SubText
             }
         }
 
-        public Image Image => Images.StaticImages._000_EnablePowerControl_h32bit_16;
+        public Image Image => Images.StaticImages._000_Module_h32bit_16;

```

Comment:
The flash icon is not correct in this case. Ideally it would be nice if we had an icon for monitoring resources, but we don't have one at the moment, so I think we can use the module icon which signifies a plugin.